### ISSUE TITLE
Upgrade angular Version 

### ISSUE THREAD

**Issue by HaithemMosbahi** [HaithemMosbahi/ngx-avatar#28]

**Comment by beeing** [comment]
May also consider upgrading to angular 6, having error below:

```
ngx-avatar/avatar.service.d.ts(3,10): error TS2305: Module '"rxjs/Observable"' has no exported member 'Observable'.
```

**Comment by tomgruszowski** [comment]
+1 on the ng6 update, might as well, rxjs compat is a sizable library to keep around.

**Comment by btastic** [comment]
Can we get this PR merged? Should do what we need: https://github.com/HaithemMosbahi/ngx-avatar/pull/29

**Comment by odahcam** [comment]
@HaithemMosbahi do you need any maintainers?

**Comment by HaithemMosbahi** [comment]
This should be fixed in version 3.0.0. It's now compatible with Angular 6.

@odahcam Yes, That would be great if you can maintain the component with me
